notNeededPackages.json

```diff
@@ -4203,6 +4203,10 @@
             "libraryName": "react-query",
             "asOfVersion": "1.2.8"
         },
+        "react-redux": {
+            "libraryName": "react-redux",
+            "asOfVersion": "8.0.0"
+        },
```

Comment:
Oh hey, yeah, the `not-needed` script does clearly nuke the folders, doesn't it :)

Reply:
Just pushed a change to include this part. Let's see if the CI is happy with that.

Reply:
CI seems fine with it. I guess we can hope future changes will correctly publish?

Reply:
Although that does mean any packages that depend on `react-redux`'s types are currently using v7 and not v8 which isn't great.

Reply:
That would be the case either way - assuming that package had a normal semver dependency, not `*".

Reply:
But going by the readme, this would even lead to an empty 8.0 package to be published.

Reply:
> That would be the case either way - assuming that package had a normal semver dependency, not `*".

Maybe I'm misunderstanding what you're saying. I believe [the recommendation](https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package) is for DefinitelyTyped packages to add a dependency on `react-redux` that would correctly resolve to v8.

Reply:
> Although that does mean any type packages on DefinitelyTyped that depend on react-redux's types are currently using v7 and not v8 which isn't great.

My point is: is something depends on `@types/react-redux`, it does so because it did so a year ago. It depends on `7.x.x` and will never install a `8.x.x`. 
That will stay that way until the packager author manually updates that to `8.0.0` - and at that point they should remove the dependency anyways.

Reply:
Yeah, absolutely, I'm just talking about the latest versions of DefinitelyTyped packages depending on `@types/react-redux` instead of `react-redux`. Normally updating those dependencies is a required part of removing a types package, but has not been done in this PR since we didn't actually delete the types. Ideally all DefinitelyTyped packages would be updated to depend on `react-redux` instead of `@types/react-redux`.

Reply:
Ah, understood!